### PR TITLE
style_fix

### PR DESCRIPTION
**Pull request by omrihecht**
style_fix

### CODE REVIEW

File: projects/ui-framework/src/lib/form-elements/input/input.component.ts [hunk 1]
```diff
@@ -98,4 +106,10 @@ export class InputComponent extends BaseFormElement implements OnInit {
       this.propagateChange(value);
     }
   }
+
+  hasTranscludeContent(node: HTMLElement): boolean {
+    return node.childNodes.length > 0
```
Comment (ishaiborovoy) [inline projects/ui-framework/src/lib/form-elements/input/input.component.ts]: I think you can do it just with return, no need if then here

Comment (omrihecht) [reply projects/ui-framework/src/lib/form-elements/input/input.component.ts]: how?

File: projects/ui-framework/src/lib/form-elements/input/input.component.ts [hunk 2]
```diff
@@ -98,4 +106,10 @@ export class InputComponent extends BaseFormElement implements OnInit {
...
+      ? node.childNodes.length > 1 || !includes(this.emptyNodeNames, node.childNodes[0].nodeName)
+      : false;
+  }
 }
```
Comment (ishaiborovoy) [inline projects/ui-framework/src/lib/form-elements/input/input.component.ts]: return node.childNodes.length > 1 && (node.childNodes.length > 1 || !includes(this.emptyNodeNames, node.childNodes[0].nodeName));

Not sure what is more readble ... So let you decide...

File: projects/ui-framework/src/lib/form-elements/input/input.component.ts (hunk 1 above)
Comment (ishaiborovoy) [reply projects/ui-framework/src/lib/form-elements/input/input.component.ts]: return node.childNodes.length > 0 && (node.childNodes.length > 1 || !includes(this.emptyNodeNames, node.childNodes[0].nodeName));

Not sure what is more readble ... So let you decide...

**Pull request closed without merging**